### ISSUE TITLE
Error 520 from Cloudflare on a simple request on users

### ISSUE THREAD

**Issue by fzaninotto** [typicode/jsonplaceholder#160]
I tried accessing this URL: https://jsonplaceholder.typicode.com/users?id=3&id=9&id=2&id=7&id=10&id=5

And the result, in the browser, is: 

![image](https://user-images.githubusercontent.com/99944/115765878-93d4e100-a3a7-11eb-9003-9f8fee4b075f.png)

When accessing this same URL through fetch, the 530 error turns into a CORS failure.

**Comment by realhefd** [comment]
I'm also getting 502 from any request I make. Looks like the actual site is down and Cloudflare is just providing us with a snapshot.

**Comment by large** [comment]
I having the same issue as reported above.
Tested some links...

Working:
https://jsonplaceholder.typicode.com/posts

Not working (520):
https://jsonplaceholder.typicode.com/albums/1
https://jsonplaceholder.typicode.com/users/
https://jsonplaceholder.typicode.com/photos

Hopefully the mockingbird will get up and running soon :)

**Comment by realhefd** [comment]
How frequently does this issue occur, because it can mess up somebody pretty bad! At least if we know the frequency we can prepare.

**Comment by jaisharx** [comment]
Yeah I can confirm this too. The [https://jsonplaceholder.typicode.com/](https://jsonplaceholder.typicode.com/) is legit down. None of the requests to the endpoints are working, and the homepage is only serving up cached version of the page.

**Comment by KoboldM** [comment]
I am also getting an error with the website. I'm sometimes even getting the heroku page that the app is down.

**Comment by ChaitanyaVSH** [comment]
The issue is happening since morning in India, I'm blocked on my pocket projects as I regularly use /users /posts end-points.

**Comment by Kashifcbt** [comment]
Is there any plan to resolve this issue or its permanently down?

**Comment by typicode** [comment]
It should be good now.
